### PR TITLE
Furry Pride with no Removal

### PR DESCRIPTION
**Pull request by chesse20**
## About The Pull Request

image
art from @MrDoomBringer 
adds this furry pride spray without removal

## Why It's Good For The Game
Same as #68077, just with out removal
image
Requested by maintainers in #68120 

## Changelog

:cl:
add: Furry Pride large spraypaint added to spraycans
/:cl:

### CODE REVIEW

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 	var/static/list/runes = list("rune1","rune2","rune3","rune4","rune5","rune6")
 	var/static/list/randoms = list(RANDOM_ANY, RANDOM_RUNE, RANDOM_ORIENTED,
 		RANDOM_NUMBER, RANDOM_GRAFFITI, RANDOM_LETTER, RANDOM_SYMBOL, RANDOM_PUNCTUATION, RANDOM_DRAWING)
-	var/static/list/graffiti_large_h = list("yiffhell", "secborg", "paint")
+	var/static/list/graffiti_large_h = list("yiffhell","furrypride", "secborg", "paint")
```
Comment (Mothblocks) [inline code/game/objects/items/crayons.dm:49]:
```suggestion
	var/static/list/graffiti_large_h = list("yiffhell", "furrypride", "secborg", "paint")
```

**Comment by Son-of-Space** [comment]
it looks too proper, especially next to the other one. You gotta rough it up a bit. People are going to be quickly spraying graffiti to make it.

**Comment by Fikou** [comment]
YEaaah GAng wars spray gang wars lets gooo boys IYPEEEEEEEEEEE

**Review by Twaticus — CHANGES_REQUESTED** [review]
the old spray tags are ugly and this is somehow even uglier with less effort. atleast have it fit the current tag, if not better.

**Comment by Farquaar** [comment]
As much as I can't wait to grief people with the furry pride spray you really need to fix that up. The colours are completely flat.

**Comment by chesse20** [comment]
image
is this enough noise/texturing on it? i reshaped some of the letters and added noise to make it less flat

**Comment by MrDoomBringer** [comment]
Try this instead, it uses the same font as the original sprite:
f_pride
Honored to support this infamous SPUF user in his PRs
image

**Comment by chesse20** [comment]
> Try this instead, it uses the same font as the original sprite: f_pride Honored to support this infamous SPUF user in his PRs image

sure looks good

**Comment by chesse20** [comment]
image

**Comment by Farquaar** [comment]
Looks decent now. I rescind my downvote and look forward to using this as a weapon of mass destruction

**Comment by moocowswag** [comment]
Looking forward to all of the over-escalation incidents over spray paint 

(Wouldnt be the first time, one round everyone was fighting over the color red or blue and it got lethal)

**Review by Twaticus — APPROVED** [review]

**Comment by optimumtact** [comment]
Radical centrism

**Pull request merged**